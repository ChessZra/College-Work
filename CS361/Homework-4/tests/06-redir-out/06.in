file file1
file file2
file file3
echo Hello >file1;echo World >file2;echo ! >file3
file file1;file file2;file file3
md5sum file1;md5sum file2;md5sum file3
cat file1;cat file2;cat file3
rm file1;rm file2;rm file3

