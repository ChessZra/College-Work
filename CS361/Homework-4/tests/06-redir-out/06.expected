cs361sh> file1: cannot open `file1' (No such file or directory)
cs361sh> file2: cannot open `file2' (No such file or directory)
cs361sh> file3: cannot open `file3' (No such file or directory)
cs361sh> cs361sh> file1: ASCII text
file2: ASCII text
file3: ASCII text
cs361sh> 09f7e02f1290be211da707a266f153b3  file1
52f83ff6877e42f613bcd2444c22528c  file2
8183aa57a23658efe7ba7aebe60816bc  file3
cs361sh> Hello
World
!
cs361sh> cs361sh> cs361sh> 
